user-guide/Reference/DataMiner_Tools/Cassandra_Cleaner.md

```diff
@@ -24,12 +24,16 @@ There are two configuration files present in the root directory of the tool:
 
 If these files are not present, generate them by running the executable with the `generate -c` arguments. As soon as the files have been generated, the tool will stop running. This allows you to first configure the files and then run the tool with the proper configuration.
 
-The most important settings are the `delete.start.time` and `delete.end.time` fields.
+### db.yaml
```

Comment:
What about settings.yaml? If all the information below is exclusively related to db.yaml, how should users configure settings.yaml?